### ISSUE TITLE
counterclockwise rotors

### ISSUE THREAD

**Issue by najlitarvan** [tsunamayo/Starship-EVO#6098]
it can be annoying to have to duplicate the sequencer gates due to rotors rotating only one direction, imo it would be nice to have the option to have them rotate the opposite dirrection, as for UI i do believe we could reuse the symetry option already present for mech use, if symetry is on and not connected to mech, it would rotate counterclockwise

**Comment by boxfoxscoot** [comment]
yeah its kinda annoying and also can fuck with mechs making their arms or legs twist around 360 degrees because the controller thinks they're backwards somehow despite them being in the right spot

**Comment by tsunamayo** [comment]
@boxfoxscoot It should  not impact mech, if you an issue please show me your blueprint. Thanks
